Core/src/Testing/TestHelpers.php

```diff
@@ -212,6 +213,9 @@ public static function generatedSystemTestBootstrap()
         // For generated system tests, we need to set GOOGLE_APPLICATION_CREDENTIALS
         // and PROJECT_ID to appropriate values
         $keyFilePath = getenv('GOOGLE_CLOUD_PHP_TESTS_KEY_PATH');
+        if (empty($keyFilePath)) {
+            exit('GOOGLE_CLOUD_PHP_TESTS_KEY_PATH must be set to run system tests.');
```

Comment:
Nit:
Should we place this as a constant somewhere? I see that this `env` is used in several places. I think if this code needs maintenance it would be easier to have all of the places using one constant and easy to change in one place rather than using a string.

Reply:
Yes absolutely. I've added your suggestion (and one of my own) in this issue so that we can clean this part up soon

https://github.com/googleapis/google-cloud-php/issues/7709